src/typings/che.d.ts

```diff
@@ -12,6 +12,8 @@
 
 declare namespace che {
 
+  export type ConfigurableMenuItem = 'administration' | 'factories' | 'getstarted' | 'organizations' | 'stacks';
```

Comment:
Why do you need this in the context of copying the login command?

`factories`, `organizations`, `stacks` - we don't event have such navigation items.

Reply:
Ok. I have changed it.